chrome/browser/android/adblock/regex_resource_type_detector.cpp

```diff
@@ -0,0 +1,51 @@
+/*
+ * This file is part of Adblock Plus <https://adblockplus.org/>,
+ * Copyright (C) 2006-present eyeo GmbH
+ *
+ * Adblock Plus is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * Adblock Plus is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Adblock Plus.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "chrome/browser/android/adblock/regex_resource_type_detector.h"
+#include <regex>
+
+static const std::regex RE_JS("\\.js$", std::regex_constants::ECMAScript | std::regex_constants::icase);
+static const std::regex RE_CSS("\\.css$", std::regex_constants::ECMAScript | std::regex_constants::icase);
```

Comment:
same for css - i can see arguments sometimes:
ex. `resource type of https://m.e1.ru/static/build/css/main.css?1007 `